pkg/Makefile

```diff
@@ -46,5 +46,11 @@ appimagetool-x86_64.AppImage:
 osc-cli-x86_64.AppImage: osc-cli.AppDir/AppRun osc-cli.AppDir osc-cli.AppDir/done appimagetool-x86_64.AppImage $(PY_TARGET)
 	./appimagetool-x86_64.AppImage osc-cli.AppDir/
 
+osc-cli-x86_64.zip: osc-cli
+	zip -r osc-cli-x86_64.zip osc-cli
+
+osc-cli:
+	docker run --rm -it -v ${PWD}/..:/src outscale/wine-pyinstaller:0.0.1 --distpath ./pkg --clean --name osc-cli osc_sdk/sdk.py
```

Comment:
I don't understand what is `outscale/wine-pyinstaller` shouldn't you add a git submodule ? maybe add a rule to clone it and build the docker ?

Reply:
Its a docker image hosted on [outscale-dev/wine-pyinstaller](https://github.com/outscale-dev/wine-pyinstaller) (which can be re-used for other projects).

No need to sub-module and build on the fly, `docker run` will fetch the image from docker hub and do the job for you.